test/suite/hook_tiered.py

```diff
@@ -300,7 +300,6 @@ def skip_test(self, test):
                 "test_stat_log01_readonly.test_stat_log01_readonly",
                 "test_stat_log02.test_stats_log_on_json_with_tables",
                 "test_txn02.test_ops",
-                "test_upgrade.test_upgrade",
```

Comment:
Why are we removing this test?

Reply:
@ddanderson I'll answer to make sure I understand. This test was on a list of tests to skip because they were know to fail. So by adding in the code fix, Don removed it from the skip list so that it will now be tested.

Reply:
Oh I see, I totally missed that 😐  Thanks!